include/deal.II/dofs/dof_accessor.h

```diff
@@ -3168,8 +3168,9 @@ namespace internal
 
             for (const auto line : accessor.line_indices())
               {
-                const bool line_orientation = line_orientations[line];
-                if (line_orientation)
+                const auto line_orientation = line_orientations[line];
+                if (line_orientation ==
+                    ReferenceCell::default_combined_face_orientation())
```

Comment:
Just in case this is a direction you wanted to go, you can now write this as follows as well:
```suggestion
                if (const auto line_orientation = line_orientations[line];
                    line_orientation ==
                      ReferenceCell::default_combined_face_orientation())
```

Reply:
Do we normally do this? I don't recall seeing a semicolon in an `if` statement before. I'd prefer to be stylistically consistent with the rest of the library.

Reply:
I'm not sure we do this in many places. I just wanted to point it out because it allows you to save one level of braces in
```
  for (auto &x : ...)
  {
     const auto y = x.something();
     if (y == ...)
       ...
  }
```
I'm not suggesting you do unless you think it's patently better.

Reply:
I don't feel that strongly about it so I'll choose to be consistent. We can always go back and convert things to do this later.

Has this always worked or is it a C++17 thing?

Reply:
I think it's a C++17 thing.

Reply:
See ` init-statement` in https://en.cppreference.com/w/cpp/language/if.